lib/napa/deprecations/application_api.rb

```diff
@@ -1,8 +1,10 @@
 module Napa
   class Deprecations
+    LINK = 'https://github.com/bellycard/napa/blob/master/lib/napa/generators/templates/scaffold/app/apis/application_api.rb'
```

Comment:
Yeah, I noticed that as I made this change, it used to work when the link was committed but not anymore. Not sure what happened to it.